application/extensions/BreadcrumbWidget/BreadcrumbWidget.php

```diff
@@ -122,13 +112,16 @@ private function buildQuestionGroupBreadCrumbs(array $breadcrumbConfigArray, arr
         $questionGroup = $breadcrumbConfigArray['oQuestionGroup'];
         $subAction = $breadcrumbConfigArray['sSubaction'];
         $question = $breadcrumbConfigArray['oQuestion'];
+        $mode = $breadcrumbConfigArray['mode'];
         if (isset($questionGroup)) {
+            $groupTitle = flattenText($questionGroup->questiongroupl10ns[$survey->language]->group_name);
```

Comment:
Must add a `nbsp` (or another string) in case of empty $questionGrou text

Reply:
Empty group treated here:
https://github.com/LimeSurvey/LimeSurvey/pull/4016/commits/d32483204256a78afe2ba290f8d5e0ab87a8ec68#diff-0493278a6e63cc6637dddfc52e9f2f85a08179bbdea056e4da35c8a38bd841f9R125